platformshconfig/config.py

```diff
@@ -321,9 +326,9 @@ def application(self):
 
         """
 
-        if not self.is_valid_platform():
+        if not self._applicationDef():
```

Comment:
`_applicationDef` is a `dict` not a method

Reply:
Hooray for tests!  Fixed typo.